nav2_system_tests/src/costmap_filters/tester_node.py

```diff
@@ -380,18 +453,22 @@ def run_all_tests(robot_tester):
     if (result):
         robot_tester.wait_for_node_active('amcl')
         robot_tester.wait_for_initial_pose()
+        result = robot_tester.wait_for_clearning_endpoints(10)
         robot_tester.wait_for_node_active('bt_navigator')
-        result = robot_tester.wait_for_filter_mask(10)
-
+        result = result and robot_tester.wait_for_filter_mask(10)
     if (result):
         result = robot_tester.runNavigateAction()
 
+    if robot_tester.test_type == TestType.KEEPOUT:
+        result = result and robot_tester.wait_for_voxel_pointcloud(10)
```

Comment:
Why not have the `wait_for_clearning_endpoints` under this statement as well to group them together?

Reply:
Actually, I can. will do that.